### PR TITLE
Enha: sentry-cli Integration Tests

### PR DESCRIPTION
**Pull request by denrase**
#skip-changelog

## :scroll: Description

Adds integration tests to the repository for the `sentry-cli` commands used by this plugin.

Run locally:

```bash
test/integration-test-server-start.sh
```

In separate process:

```bash
export ANDROID_HOME=~/Library/Android/sdk/
export SENTRY_URL=http://127.0.0.1:8000
./gradlew -p plugin-build test --tests SentryPluginIntegrationTest
test/integration-test-server-stop.sh
```

## :bulb: Motivation and Context

- We want to make sure that the bundled `sentry-cli` version can run the used commands without issue.
- This is still very much a work in progress, as it is not clear yet how to best run there tests on CI

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [ ] I updated the docs if needed
- [x] No breaking changes

## :crystal_ball: Next steps

- [x] Run test with CI

### CODE REVIEW

**Review by adinauer — APPROVED** [review]

**Comment by adinauer** [comment]
@marandaneto do you also want to give this a review or can I just merge it?

File: .github/workflows/integration-tests-sentry-cli.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: integration-tests-sentry-cli
+
+on:
+  push:
+    branches:
+      -master
```
Comment (marandaneto) [inline .github/workflows/integration-tests-sentry-cli.yml:6]:
```suggestion
      - main
      - release/**
```

**Comment by marandaneto** [comment]
> @marandaneto do you also want to give this a review or can I just merge it?

Feel free to merge it when there's one approval.

**Review by marandaneto — APPROVED** [review]
Thanks @denrase

**Pull request merged**